recipes/trio-websocket/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set version = "0.9.2" %}
+
+package:
+  name: trio-websocket
+  version: {{ version }}
+
+source:
+  - folder: dist
+    url: https://pypi.io/packages/source/t/trio-websocket/trio-websocket-{{ version }}.tar.gz
+    sha256: a3d34de8fac26023eee701ed1e7bf4da9a8326b61a62934ec9e53b64970fd8fe
+  - folder: src
+    url: https://github.com/HyperionGray/trio-websocket/archive/refs/tags/{{ version }}.tar.gz
+    sha256: acc72db00e89e4594caac269aabebeccbd7b2d87d4dd06acba7adf4344cd8a42
+
+build:
+  number: 0
+  noarch: python
+  script: cd dist && {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.5
+  run:
+    - async_generator >=1.10
+    - python >=3.5
+    - trio >=0.11
+    - wsproto >=0.14
+
+test:
+  source_files:
+    - src/pytest.ini
+    - src/tests
+  imports:
+    - trio_websocket
+  commands:
+    - pip check
+    - cd src && pytest -vv --cov trio_websocket --cov-report term-missing:skip-covered --no-cov-on-fail --cov-fail-under 95
+  requires:
+    - attrs >=19.2.0
+    - pip
+    - pytest >=4.6
+    - pytest-cov
+    - pytest-trio >=0.5.0
+    - trustme
+
+about:
+  home: https://github.com/HyperionGray/trio-websocket
+  summary: WebSocket library for Trio
+  license: MIT
+  # TODO: revisit if/after merge of https://github.com/HyperionGray/trio-websocket/pull/168
```

Comment:
This looks merged now. Does the  comment still apply?

Everything else looks spot-on!

Reply:
Yes, was merged, but hasn't been released.